cosmos/Dockerfile

```diff
@@ -3,6 +3,8 @@ ARG COSMOS_TAG=latest
 
 FROM ${COSMOS_REGISTRY}/ballaerospace/cosmosc2-ruby:${COSMOS_TAG}
 
+ENV RUBYLIB=/cosmos/lib
+ENV COSMOS_PATH="/cosmos"
```

Comment:
Couldn't get it to use my locally built code without this. It kept going out to Rubygems and pulling the latest gem.

Reply:
The idea here was the gem is installed in cosmosc2-ruby now, so these shouldn't be necessary.  (If they are something is wrong)